scripts/generator/graphql_generator/csharp/SDK/iOS/iOSNativeCheckout.ApplePayEventReceiver.cs.erb

```diff
@@ -29,16 +30,35 @@ namespace <%= namespace %>.SDK.iOS {
             var contentDictionary = (Dictionary<string, object>)Json.Deserialize(message.Content);
             var mailingAddressInput = new MailingAddressInput(contentDictionary);
 
-            List<SummaryItem> summaryItems = null;
-            List<ShippingMethod> shippingMethods = null;
+            Action respondWithSuccess = () => {
+                message.Respond(new ApplePayEventResponse(ApplePayAuthorizationStatus.Success, GetSummaryItems(), GetShippingMethods()).ToString());
+            };
+
+            Action<ShopifyError> respondToError = (ShopifyError error) => {
+                switch(error.Type) {
+                case ShopifyError.ErrorType.GraphQL:
```

Comment:
This case should fall through to `ShopifyError.ErrorType.HTTP`, which is just a failure to complete `ApplePay`